libbeat/cmd/instance/beat.go

```diff
@@ -147,6 +148,7 @@ func NewBeat(name, indexPrefix, v string) (*Beat, error) {
 			Hostname:    hostname,
 			UUID:        uuid.NewV4(),
 		},
+		Fields: assets.GetFields(),
```

Comment:
Maybe the beat name could be a param? (thinking of two beats in the same process)

Reply:
Are you thinking it cause some issues with the collectbeat from @vjsamuel ? Did not think about that TBH.